rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -271,17 +277,19 @@ pub trait HasPrometheus {
 }
 
 impl HasPrometheus for TestEnv {
-    fn sync_with_prometheus(&self) {
-        self.sync_with_prometheus_by_name("", None)
+    fn sync_with_prometheus(&self, boundary_node_name: &str) {
```

Comment:
Wouldn't an optional Boundary Node name make more sense here instead of passing the empty string in case there's no Boundary Node?
```suggestion
    fn sync_with_prometheus(&self, boundary_node_name: Option<String>) {
```

Reply:
I would go even further and have two methods, one which accepts `boundary_node_name` and one which doesn't

Reply:
Done.